network-api/networkapi/wagtailpages/factory/image_factory.py

```diff
@@ -0,0 +1,24 @@
+import factory
+from wagtail.images import get_image_model
+from wagtail.core.models import Collection
+
+
+# Slightly modify the wagtail_factories ImageFactory so that it
+# always generates images in the Root collection:
+
+class CollectionMemberFactory(factory.DjangoModelFactory):
+    collection = Collection.objects.get(name='Root')
```

Comment:
wagtail_factories uses `collection = factory.SubFactory(CollectionFactory, parent=None)` here

Reply:
I dont see any more collections in localhost. Nicely done!